setup.py

```diff
@@ -100,10 +100,11 @@
     'Topic :: Software Development'
   ],
   install_requires = ['six>=1.11.0', 'subprocess32; python_version < "3"',
-      'python-dateutil>=2.8.0'],
+      'python-dateutil>=2.8.0', 'asn1crypto; python_version > "3"'],
```

Comment:
I mentioned this elsewhere but this seems wrong: it should be only needed for "pkcs11" extra, right?